src/components/form/question/mosaic/mosaicQuestion/MosaicNumberInput.tsx

```diff
@@ -23,7 +23,13 @@ export default function NumberInput({
   parentMosaic,
   ...props
 }: Props) {
-  const { value, isMissing } = useRule(question)
+  const { value, isMissing, plafond } = useRule(question)
+
+  const isPlusDisabled =
+    value !== undefined &&
+    typeof value === 'number' &&
+    plafond !== undefined &&
+    value === plafond
```

Comment:
You need typeguards for all your checks

Reply:
You mean I need to check the `plafond` type as well ?

Reply:
You mean I need too check for `plafond` type as well ?